modules/Cargo.toml

```diff
@@ -108,7 +108,7 @@ abstract-client = { path = "../framework/packages/abstract-client" }
 rpath = false
 lto = true
 overflow-checks = true
-opt-level = 3
```

Comment:
Can't we just do size optimization for the dex module specifically? We should optimize for speed in the others IMO

Reply:
> Can't we just do size optimization for the dex module specifically? We should optimize for speed in the others IMO

Yeah we can move release profile to each individual contract from the workspace

Reply:
It's not possible to override release profile if it's specified in the workspace

Reply:
> > Can't we just do size optimization for the dex module specifically? We should optimize for speed in the others IMO
> 
> Yeah we can move release profile to each individual contract from the workspace

This was actually something that Oak security told us to do in their audit report. See: https://github.com/oak-security/audit-reports/blob/master/Abstract/2023-05-09%20Audit%20Report%20-%20Abstract%20Module%20Bases%2C%20Proxy%20and%20Manager%20Contracts%20v1.0.pdf